index.html

```diff
@@ -0,0 +1,171 @@
+            <li class="account__list"><a href="./fashion.html" class="account__link">Fashion</a></li>
+            <li class="account__list"><a href="./register.html" class="account__link">Register</a></li>
+            <li class="account__list"><a href="./login.html" class="account__link"><i class="fa-solid fa-right-to-bracket"></i>  Login</a></li>
+            <li class="account__list"><a href="./shoppingcart.html" class="account__link"><i class="fa-solid fa-cart-shopping"></i>  Cart</a></li>
+        </ul>
+        <div class="account-bar">
+            <i class="fa-solid fa-bars"></i>
+        </div>
+    </nav>
+     <!-- User account creation ends here-->
+
+    <!-- main starts here-->
+    <main>
+        <section class="banner">
+            <div class="banner__description">
+                <h2 class="banner__title">New Arrivals for Women and Men</h2>
+                <p class="para__description">Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+                    Molestias eveniet sint commodi tempora 
+                    consequatur cumque sit! Nobis deleniti labore veritatis.
+                    <span>New</span>
+                </p>
+                <button class="btn btn-learn" aria-label="learn more about fashion">Show More</button>
+            </div>
+           <figure class="banner__img">
+            <img class="img" src="images/Banner-images/image3.jpg" alt="product-wearing-woman">
+           </figure>
+        </section>
+
+        <!-- product details with payment details starts here-->
+        <section class="product">
+            <h1 class="product__title">Our Latest Products</h1>
+            <div class="product__product">
+                <article class="product__card">
+                    <img src="images/Skirts-image/skirt1.jpeg" alt="skirt-image" class="card__img">
+                    <span  aria-label="for description of the article to whom it belongs" class="card__description">Women</span>
+                    <span  aria-label="brand specifier and product description" class="card__brand">Lady skirt</span>
+                    <div class="payment">
+                        <span class="payment__detail">$60</span>
+                        <button class="btn btn-price"><a href="./productDetail.html">Buy</a></button>
+                    </div>
+                </article>
+
+                <article class="product__card">
+                    <img src="images/Skirts-image/sweater1.jpg" alt="sweater-image" class="card__img">
+                    <span  aria-label="for description of the article to whom it belongs" class="card__description">Men/Women</span>
+                    <span  aria-label="brand specifier and product description" class="card__brand">Man/Women Sweater</span>
+                    <div class="payment">
+                        <span class="payment__detail">$40</span>
+                        <button class="btn btn-price"><a href="./productDetail.html">Buy</a></button>
```

Comment:
can you style the button and a to have better contrast?

Reply:
Ok.